doc/user/metrics/service-monitor.md

```diff
@@ -0,0 +1,45 @@
+## Using the ServiceMonitor prometheus-operator CRD
+
+[prometheus-operator][prom-operator] an operator that creates, configures and manages Prometheus clusters atop Kubernetes.
+
+`ServiceMonitor` is a [CR][cr] of the prometheus-operator, which discovers the `Endpoints` in `Service` objects and configures Prometheus to monitor those `Pod`s. See the prometheus-operator [documention][sm] to learn more about `ServiceMonitor`s.
```

Comment:
Typo. Also change the link tag from sm to service-monitor.
Mostly for our sake but at a glance it's more clear what we're linking to.
```suggestion
`ServiceMonitor` is a [CR][cr] of the prometheus-operator, which discovers the `Endpoints` in `Service` objects and configures Prometheus to monitor those `Pod`s. See the prometheus-operator [documentation][service-monitor] to learn more about `ServiceMonitor`s.
```